### DISCUSSION TITLE
Add gender icons

### DISCUSSION THREAD

**Discussion by seb-jean** [tailwindlabs/heroicons#724] [Ideas]
Hi,
It would be cool to have gender icons :)
Examples:

- https://icons.getbootstrap.com/icons/gender-ambiguous/
- https://icons.getbootstrap.com/icons/gender-female/
- https://icons.getbootstrap.com/icons/gender-male/
- https://icons.getbootstrap.com/icons/gender-trans/

**Comment by erikgerrits** [comment]
I'm using the icon set in [elon.io](https://elon.io), and have everything I need except for those gender icons.

**Comment by good3n** [comment]
+1

**Comment by InsomniakDev** [comment]
+1

**Comment by RanaMoizHaider** [comment]
+1
